components/script/dom/htmlmetaelement.rs

```diff
@@ -73,6 +73,7 @@ impl HTMLMetaElement {
                         rules: vec![CSSRule::Viewport(translated_rule)],
                         origin: Origin::Author,
                         media: None,
+                        depends_on_viewport_size: true,
```

Comment:
On second thought, maybe this stylesheet (one that contains a generated `@viewport` rule) should have the flag set to `false`, since viewport constraints are already recalculated on resize and shouldn't require the whole page to be recomputed.